src/coreComponents/constitutive/permeability/permeabilitySelector.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file permeabilitySelector.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_PERMEABILITY_PERMEABILITYSELECTOR_HPP
+#define GEOSX_CONSTITUTIVE_PERMEABILITY_PERMEABILITYSELECTOR_HPP
+
+#include "constitutive/ConstitutivePassThruHandler.hpp"
+#include "ConstantPermeability.hpp"
+#include "CarmanKozenyPermeability.hpp"
+#include "ParallelPlatesPermeability.hpp"
+#include "StrainDependentPermeability.hpp"
+
+namespace geosx
+{
+
+namespace constitutive
+{
+
+#ifndef PASSTHROUGH_HANDLE_CASE
+#define PASSTHROUGH_HANDLE_CASE( MODEL ) \
+  if( dynamicCast< MODEL * >( &perm ) ) \
+  { \
+    lambda( static_cast< MODEL & >( perm ) ); \
+    return true; \
+  }
```

Comment:
so, I created this in the other PR in which the permeability models are already used. I think we will need a dispatch but it really depends on what the answer to my question about the permeability is. It may very well be that the entire file is useless.

Reply:
Actually, the `PASSTHROUGH_HANDLE_CASE` macro seems unused? Am I right?

Reply:
It's a remnant of older switch code we had in fluid models. It was replaced by `ConstitutivePassThruHandler<>` template (which I recommend using, unless you like typing if/else branches) and was left in `relativePermeabilitySelector.hpp` by accident.